lib/src/_shared/ModalDialog.tsx

```diff
@@ -38,76 +63,56 @@ export const ModalDialog: React.SFC<ModalDialogProps & WithStyles<typeof styles>
   showTabs,
   wider,
   ...other
-}) => (
-  <Dialog
-    role="dialog"
-    onClose={onDismiss}
-    classes={{
-      paper: clsx(classes.dialogRoot, {
-        [classes.dialogRootWider]: wider,
-      }),
-    }}
-    {...other}
-  >
-    <DialogContent children={children} className={classes.dialog} />
-
-    <DialogActions
+}) => {
+  const classes = useStyles();
+  return (
+    <Dialog
+      role="dialog"
+      onClose={onDismiss}
       classes={{
-        root: clsx({
-          [classes.withAdditionalAction]: clearable || showTodayButton,
+        paper: clsx(classes.dialogRoot, {
+          [classes.dialogRootWider]: wider,
         }),
       }}
+      {...other}
     >
-      {clearable && (
-        <Button color="primary" onClick={onClear}>
-          {clearLabel}
-        </Button>
-      )}
+      <DialogContent children={children} className={classes.dialog} />
 
-      {showTodayButton && (
-        <Button color="primary" onClick={onSetToday}>
-          {todayLabel}
-        </Button>
-      )}
+      <DialogActions
+        classes={{
+          root: clsx({
+            [classes.withAdditionalAction]: clearable || showTodayButton,
+          }),
+        }}
+      >
+        {clearable && (
+          <Button color="primary" onClick={onClear}>
+            {clearLabel}
+          </Button>
+        )}
 
-      {cancelLabel && (
-        <Button color="primary" onClick={onDismiss}>
-          {cancelLabel}
-        </Button>
-      )}
+        {showTodayButton && (
+          <Button color="primary" onClick={onSetToday}>
+            {todayLabel}
+          </Button>
+        )}
 
-      {okLabel && (
-        <Button color="primary" onClick={onAccept}>
-          {okLabel}
-        </Button>
-      )}
-    </DialogActions>
-  </Dialog>
-);
+        {cancelLabel && (
```

Comment:
I'm wondering about a new name for the localization wordings (for consistency, as detailed above).

```suggestion
        {cancelText && (
```

Reply:
Let's pull this stuff out to separate PR with `localization support`. This one is so giant

Reply:
Yes, I agree!